api/v1beta1/openstackconfiggenerator_types.go

```diff
@@ -60,6 +60,25 @@ type OpenStackConfigGeneratorSpec struct {
 	EnableFencing bool `json:"enableFencing"`
 	// TripleoRoleOverride - map of TripleO role name to temporary role override to support a multi-rhel environment (valid for 17.1 only)
 	TripleoRoleOverride map[string]TripleoRoleOverrideSpec `json:"tripleoRoleOverride,omitempty"`
+	// +kubebuilder:validation:Optional
+	Advanced OpenStackConfigGeneratorAdvancedSettings `json:"advanced,omitempty"`
```

Comment:
To make it clearer that this isn't for regular users I would name it "debug" or "devMode"